code/client/src/components/SiderMenu.jsx

```diff
@@ -27,16 +32,27 @@ const SiderMenu = ({ ...args }) => {
   const nav = ({ key }) => {
     history.push(Paths[key])
   }
+  const [stats, setStats] = useState(null)
+
+  useEffect(() => {
+    async function getStats () {
+      const statsData = await util.getWalletStats()
+      setStats(statsData)
+    }
+    getStats()
```

Comment:
need to do setInterval here and clearInterval on exit, since a lot of people just leave the browser tab open and this would only run once

Reply:
@polymorpher I thought about adding this but the stats are only updated once every 24 hours on the backend.